platform/linux-generic/odp_pool.c

```diff
@@ -1258,6 +1258,10 @@ int odp_pool_info(odp_pool_t pool_hdl, odp_pool_info_t *info)
 		info->dma_pool_param.uarea_size = pool->params.buf.uarea_size;
 		info->dma_pool_param.cache_size = pool->params.buf.cache_size;
 
+	} else if (pool->type_2 == ODP_POOL_ML_COMPL) {
```

Comment:
Seems like implementations are missing for event user area APIs: `odp_event_user_area()`, `odp_event_user_area_and_flag()`

Reply:
`test_ml_compl_user_area()` could also test there functions.